docs/pages/cheatsheet.adoc

```diff
@@ -716,17 +793,8 @@ A group can contain only menu link or menu table elements.
 [options="header", cols="1,1,2"]
 |====================
 |Parameter name | Parameter type        | Meaning
+|choices        | Row declaration + expression  | The possible values for selection.
```

Comment:
We have 2 tables directly one after the other. I can't figure out which one is for what. If the first one is for the "view type" menu, then we should put it directly under the syntax IMO.

Reply:
this chapter is reorganized